web/Web/web/core/session_utils.py

```diff
@@ -52,6 +57,69 @@ def submit_new_session_form(request):
         request.user.current_session = session
         request.user.save()
 
+        if 'topic-judgments_file' in request.FILES:
+            judgments_dict, msg_type, msg = handle_judgments_file(request.FILES['topic-judgments_file'])
+
+            if judgments_dict:
+                Judgment.objects.bulk_create([
+                    Judgment(
+                        user=request.user,
+                        session=session,
+                        doc_id=doc_id,
+                        doc_title=doc_title,
+                        relevance=relevance,
+                        source="seed",
+                        historyVerbose=[{
+                            "username": request.user.username,
+                            "source": "seed",
+                            "judged": True,
+                            "relevance": relevance
+                        }]
+                    ) for doc_id, (doc_title, relevance) in judgments_dict.items()
+                ])
+                request.user.save()
+            elif msg_type == messages.ERROR:
+                msg += " Session deleted."
+                session.delete()
+            messages.add_message(request, msg_type, msg)
+
+        session.begin_session_in_cal()
+
+
+def handle_judgments_file(file):
+    try:
+        if not file.name.endswith('.csv'):
+            return None, messages.ERROR, 'Please upload a file ending with .csv extension.'
+
+        judgments_file = TextIOWrapper(file.file, encoding='utf-8')
```

Comment:
Minor edit. Let's just use `io.TextIOWrapper` since we imported the whole module. And then remove the `from io import TextIOWrapper` from the imports.

Reply:
If you have `isort` in your machine, run `/usr/local/bin/isort -sl -e -fss -m 4 -w 79 <filename>` to sort the imports.